modules/abstract-utxo/src/transaction/fixedScript/verifyTransaction.ts

```diff
@@ -100,7 +122,7 @@ export async function verifyTransaction<TNumber extends bigint | number>(
   const missingOutputs = parsedTransaction.missingOutputs;
   if (missingOutputs.length !== 0) {
     // there are some outputs in the recipients list that have not made it into the actual transaction
-    throw new Error('expected outputs missing in transaction prebuild');
+    throwTxMismatch('expected outputs missing in transaction prebuild');
```

Comment:
this is plausibly an actual intent mismatch

Reply:
agree